packages/sakuli-legacy/src/context/sahi/assertion/assertion-api.function.ts

```diff
@@ -83,24 +94,45 @@ export function assertionApi(
     element: SahiElementQueryOrWebElement,
     message?: string
   ): Promise<void> {
+    if (!message) {
+      message = `The given element "${await getElementText(
+        element
+      )}" does not exist`;
```

Comment:
As the expected element could also be an image or something like that, I'd suggest to not grab the text but instead log an element selector for example.